### PR TITLE
Yaledoorman

### PR DESCRIPTION
**Pull request by jonasinghammar**
Fixed two small issues in version 0.1.4:

- When arming or disarming there was an error message stating 'No alarm code set'
- When adding multiple Yale Doorman locks the status for all locks was set to the last added lock

I'm not a JavaScript developer so please feel free to reject this Pull Request and implement the changes in a proper way :)

I've been running this code on my Homey for a few hours now so far it works like a charm!
Thank you very much for really good work :)

### CODE REVIEW

**Review by Vaggan — COMMENTED** [review]
If code is null or undefined we still want to reject with an error message.
I think it should be:
`if (!code || code === '') {`

File: drivers/sector_alarm_central/device.js [hunk 1]
```diff
@@ -316,7 +316,7 @@ class MyDevice extends Homey.Device {
   async setAlarmState(state, action) {
     this.homey.app.updateLog('Function setAlarmState start', 2);
     return new Promise((resolve, reject) => {
-      if (code || code === '') {
+      if (code === '') {
```
Comment (Vaggan) [inline drivers/sector_alarm_central/device.js:319]: If code is null or undefined we still want to reject with an error message.
I think it should be:
`if (!code || code === '') {`

Comment (jonasinghammar) [reply drivers/sector_alarm_central/device.js:319]: Yes of course there should be a check for null or undefined.  

```suggestion
     if (!code || code === '') {
```

**Review by Vaggan — APPROVED** [review]

**Comment by Vaggan** [comment]
Thanks for your contribution and donation, much appreciated

**Pull request merged**